site/gr/index.html

```diff
@@ -0,0 +1,672 @@
+<!--
+  ~ Copyright (c) 2019 Goldman Sachs and others.
+  ~ All rights reserved. This program and the accompanying materials
+  ~ are made available under the terms of the Eclipse Public License v1.0
+  ~ and Eclipse Distribution License v. 1.0 which accompany this distribution.
+  ~ The Eclipse Public License is available at http://www.eclipse.org/legal/epl-v10.html
+  ~ and the Eclipse Distribution License is available at
+  ~ http://www.eclipse.org/org/documents/edl-v10.php.
+  -->
+
+<!DOCTYPE html>
+
+<html lang="en">
+
+<head>
+
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta name="description" content="">
+    <meta name="author" content="">
+    <link rel="shortcut icon" type="image/x-icon" href="favicon.ico" />
+
+    <title>Eclipse Collections - Χαρακτηριστικά που θέλετε, με τα collections που χρειάζεστε. </title>
```

Comment:
I left it as such on purpose, as otherwise, the focus of the reader will not be on the Java Collections framework. I think we would use it as such in spoken language, too. Else, the word "συλλογές" is misleading (I would personally think of some music collections, as a reader) in a Java context.  @martidis WDYT on this one?

Reply:
I think it makes sense leaving it untranslated.

Reply:
It feels a bit strange for me, the frequent mix of english terms among the greek text. I do  not read documentation in greek, so it might be the norm. imo a reader of such document should thing of java collections when reads "συλλογές". If you decide to follow this change, I would also change Lists to "Λίστες", etc.

Reply:
Cool, will be left as is.